src/components/Operations.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { withRouter } from "react-router-dom";
+
+class Operations extends Component {
+
+    constructor() {
+        super()
+        this.state ={
+            amount: "",
+            vendor: "",
+            Category: ""
+        }
+    }
+
+    deposit = () => {
+      if(this.state.amount) {
+        this.props.deposit(parseInt(this.state.amount),this.state.vendor,this.state.Category)
+        this.props.history.push("/transactions");
+      }
+    }
+
+    withraw = () => {
+      if(this.state.amount) {
+        this.props.withdraw(parseInt(this.state.amount),this.state.vendor,this.state.Category)
+        this.props.history.push("/transactions");
+      }
+
+        
+    }
+
+    updateAmount = (event) => {
```

Comment:
use only one input handler you do not need one for each input. Add a name attribute to your inputs and grab that name by doing 
`event.target.name` then use that name as the key to save it to state